README.md

```diff
@@ -9,16 +9,38 @@ Meteor-electron strives to be the easiest way to create a desktop Meteor applica
 
 `meteor add quark:electron`
 
-After adding the package, an electron app wrapper will be while you develop.
+After adding the package, an electron app wrapper will be built and opened.
+
+This app, as well as the ready-to-distribute binaries (see [Deploy](#deploy)), is built within
+`YOUR_PROJECT_DIRECTORY/.meteor-electron`. This allows the apps to be easily located as well as
+(forthcoming) the apps to be cached for speedier startup. You should add this directory to your
+`.gitignore`.
 
 ## Configuration
 Limited configuration is possible via `Meteor.settings.electron` For example
 ```json
 {
   "electron": {
+    "name": "MyApp",
+    "icon": {
+      "osx": "build/MyApp.icns"
+    },
+    // Must conform to Semver: https://docs.npmjs.com/getting-started/semantic-versioning.
```

Comment:
Good idea. Let's tackle that when the README as a whole is polished for distribution: https://github.com/rissem/meteor-electron/issues/9#issuecomment-161782284